config.example.yaml

```diff
@@ -57,5 +57,11 @@ council_changelog: 1234567890
 required_difference: 5  # Majority required for this blob to move into the next stage
 required_votes: 15  # Minimum amount of total votes before moving to next stage
 
-# Versus voting message type, Verbose (true) for a more descriptive voting message, or Compact (false) for the standard emoji vs emoji
+# Versus voting message type, Verbose (true), or Compact (false)
+# Verbose style:
+# [1] em1
+#    v
+# [2] em2
+# Compact style:
+# em1 v em2
```

Comment:
I almost feel like this one is too verbose - should aim for an explanation over example for brevity - but at this point it's not enough to make me block the PR. I can fix this later.